test/functional/page_objects/share_page.ts

```diff
@@ -45,9 +45,7 @@ export class SharePageObject extends FtrService {
    * in a pure OSS environment, the permalinks sharing panel is displayed initially
    */
   async openPermaLinks() {
-    if (await this.testSubjects.exists('sharePanel-Permalinks')) {
-      await this.testSubjects.click(`sharePanel-Permalinks`);
-    }
+    await this.openShareMenuItem(`Permalinks`);
```

Comment:
The other way of opening was really flaky when testing multiple quick shares in succession, so switching over to the more robust `openShareMenuItem` fixed this 👍